src/test/java/gov/healthit/chpl/aqa/stepDefinitions/Base.java

```diff
@@ -95,6 +95,38 @@ public void checkIfFileIsDownloaded(final String fileName)throws FileNotFoundExc
         }
     }
 
+    /**
+     * Checks if the file contains all the data and is downloaded completely or not.
+     * @param fileName expected downloaded file name
+     * @throws FileNotFoundException if the expected file not found
+     */
+    public void checkCompleteFileDownload(final String fileName) throws FileNotFoundException {
+        boolean completeFileDwnld = true;
```

Comment:
Could you just write out the word "download" in this variable name? I don't feel like leaving a few letters out is advantageous for any particular reason.